docs/source/status.rst

```diff
@@ -44,9 +44,9 @@ Data Types
 +-------------------+-------+-------+-------+----+-------+-------+-------+-------+-----------+
 | Float32/64        | ✓     | ✓     | ✓     | ✓  |  ✓    |  ✓    | ✓     | ✓     | ✓         |
 +-------------------+-------+-------+-------+----+-------+-------+-------+-------+-----------+
-| Decimal32         | ✓     |       | ✓     |    |  ✓    |       |       |       | ✓         |
+| Decimal32         | ✓     |       | ✓     |    |  ✓    |  ✓    |       |       | ✓         |
```

Comment:
See 
- https://docs.rs/arrow/latest/arrow/array/types/struct.Decimal32Type.html
- https://docs.rs/arrow/latest/arrow/array/type.Decimal32Array.html
(and similarly for Decimal64)

This was courtesy of @CurtHagenlocher  via ❤️ 
- https://github.com/apache/arrow-rs/pull/7841